spk/ffsync/Makefile

```diff
@@ -34,11 +34,6 @@ ADMIN_PORT = $(SERVICE_PORT)
 
 CONF_DIR         = src/conf/
 
-# [mysqlclient]
-DEPENDS += cross/mysql-connector-c cross/mariadb-connector-c
-ENV += MYSQLCLIENT_CFLAGS="$(CFLAGS) -I$(STAGING_INSTALL_PREFIX)/include/mysql -I$(STAGING_INSTALL_PREFIX)/include/mariadb -I$(STAGING_INSTALL_PREFIX)/$(PYTHON_INC_DIR)"
-ENV += MYSQLCLIENT_LDFLAGS="$(LDFLAGS)"
-
```

Comment:
don't we need this to build `mysqlclient==2.1.1` wheel?

Reply:
Not sure if this should be in the `spk` Makefile or the `cross` Makefile, but I've put it back in

Reply:
So I've decided to leave this out because the spk makefile already contains:

```
BUILD_DEPENDS = cross/curl cross/mysql-connector-c
```

However, I have moved the `ENV` variables to the cross makefile.

Reply:
I think I understand you. I've removed the `BUILD_DEPENDS` from the spk Makefile and added the code block above to the cross Makefile. I only included curl originally based on this https://github.com/SynoCommunity/spksrc/issues/5939#issuecomment-1826698525, but you are probably right that it's not needed. Let's see if the errors change with the build from this commit.

Reply:
@hgy59, in terms of self contained cross packages, I know you said that python was an exception. Just to confirm that this line should stay in the spk and not also move to the cross Makefile:

```
WHEELS = src/requirements-crossenv.txt src/requirements-pure.txt
```